ci-configs/packages-preview.json

```diff
@@ -67,7 +67,10 @@
       "name": "@azure/communication-chat@next"
     },
     {
-       "name": "@azure/communication-administration@next"
+      "name": "@azure/communication-administration@next"
+    },
+    {
+      "name": "@azure/eventgrid@next"
```

Comment:
It is interesting that this change was not already in place. This is not the first preview package we are shipping for Event Grid

Reply:
Ah, looks like the PR for the previous preview was not merged: https://github.com/MicrosoftDocs/azure-docs-sdk-node/pull/840

@ellismg I would recommend we close the other PR and ensure this PR is merged at the earliest